src/sentry/static/sentry/app/components/lazyLoad.jsx

```diff
@@ -59,31 +59,26 @@ class LazyLoad extends React.Component {
 
   getComponentGetter = () => this.props.component || this.props.route.componentPromise;
 
+  handleFetchError = error => {
+    // eslint-disable-next-line no-console
+    console.error(error);
```

Comment:
along with endpoint fetching errors, it would be nice to use tags for these to be able to sift through them more easily or add special alerting rules. (even if just for the sake of dogfooding different sentry features and flows). just a thought

Reply:
That's a good idea, let's talk offline about how you think we should approach this